pyproject.toml

```diff
@@ -11,7 +11,11 @@ fastapi = "^0.112.0"
 uvicorn = {extras = ["standard"], version = "^0.30.6"}
 pydantic = "^2.8.2"
 pydantic-settings = "^2.4.0"
-
+pytest = "^8.3.3"
+assertpy = "^1.1"
+allure-pytest = "^2.13.5"
+pytest-cov = "^5.0.0"
+httpx = "^0.27.2"
```

Comment:
@vasylboliuk could you please move these dependancies related to the tests to tests group. JFYI you could install dependencies to different groups just by adding it like `poetry add <dependency> -G <groupname>`